tests/Mover.test.tsx

```diff
@@ -3145,3 +3146,366 @@ describe("Mover with virtual children provided by getParent()", () => {
+            .pressDown()
+            .activeElement((el) => {
+                expect(el?.textContent).toEqual("Button2");
+            })
+            .pressDown()
+            .activeElement((el) => {
+                expect(el?.textContent).toEqual("Button1");
+            })
+            .pressRight()
+            .activeElement((el) => {
+                expect(el?.textContent).toEqual("Button3Button4");
+            })
+            .pressDown()
+            .activeElement((el) => {
+                expect(el?.textContent).toEqual("Button3Button4");
+            })
+            .pressUp()
+            .activeElement((el) => {
+                expect(el?.textContent).toEqual("Button3Button4");
+            })
+            .pressRight()
+            .activeElement((el) => {
+                expect(el?.textContent).toEqual("Button1Button2");
+            })
+            .pressUp()
+            .activeElement((el) => {
+                expect(el?.textContent).toEqual("Button2");
+            })
+            .pressUp()
+            .activeElement((el) => {
+                expect(el?.textContent).toEqual("Button1");
+            })
+            .pressTab()
+            .activeElement((el) => {
+                expect(el?.textContent).toEqual("After");
+            })
+            .pressTab(true)
+            .activeElement((el) => {
+                expect(el?.textContent).toEqual("Button1Button2");
+            });
+    });
+
+    it("should move between more levels of connected Movers", async () => {
+        await new BroTest.BroTest(
+            (
+                <div {...getTabsterAttribute({ root: {} })}>
+                    <div
+                        {...getTabsterAttribute({
+                            mover: {
+                                direction: MoverDirections.Horizontal,
+                                connected: MoverConnections.Child,
+                            },
+                        })}
+                    >
+                        <div
+                            tabIndex={0}
+                            {...getTabsterAttribute({
+                                groupper: {},
+                                mover: {
+                                    direction: MoverDirections.Vertical,
+                                    connected: MoverConnections.All,
+                                },
+                            })}
+                        >
+                            <div
+                                tabIndex={0}
+                                {...getTabsterAttribute({
+                                    groupper: {},
+                                    mover: {
+                                        direction: MoverDirections.Horizontal,
+                                        connected: MoverConnections.Parent,
+                                    },
+                                })}
+                            >
+                                <button>Button1</button>
```

Comment:
Please update button names to be a bit more specific to their level of nesting

i.e. Button1L3